addons/account/views/account_tax_views.xml

```diff
@@ -149,10 +149,10 @@
                         <page string="Definition" name="definition">
                             <div attrs="{'invisible': [('amount_type', '=', 'group')]}">
                                 <field name="country_code" invisible="1"/>
-                                <label for="invoice_repartition_line_ids"/>
-                                <field name="invoice_repartition_line_ids"/>
-                                <label for="refund_repartition_line_ids"/>
-                                <field name="refund_repartition_line_ids"/>
+                                <label for="invoice_repartition_line_ids" class="mb-4"/>
```

Comment:
To check with a PO: don't we prefer using
```xml
<group string="Distribution for Invoices">
    <field name="invoice_repartition_line_ids" nolabel="1" colspan="2"/>
</group>
<group string="Distribution for Refunds">
    <field name="refund_repartition_line_ids" nolabel="1" colspan="2"/>
</group>
```
It's more like we used to do before (separator for the label) but we lose the interrogation mark with the tooltip (but it's useless anyway). 

The full-width-o2m bugs me anyway, but that's generic and shouldn't be addressed here

Reply:
Will try tomorrow to do that, and ask a PO which one they prefer then 👍